charts/v2.0/telemetry-metrics-filter-go/values.yaml

```diff
@@ -0,0 +1,109 @@
+---
+global:
+  appVersion: 0.1.0-20221128191357.bb76027
+
+image:
+  repository: artifactory.algol60.net/csm-docker/unstable/telemetry-metrics-filter
```

Comment:
Once https://github.com/Cray-HPE/telemetry-metrics-filter/pull/4 is merged need to change `unstable` to `stable`